codec/decoder/core/src/cabac_decoder.cpp

```diff
@@ -145,6 +145,9 @@ int32_t DecodeBinCabac (PWelsCabacDecEngine pDecEngine, PWelsCabacCtx pBinCtx, u
   int32_t iRenorm = 1;
   uint32_t uiRangeLPS = g_kuiCabacRangeLps[uiState][ (uiRange >> 6) & 0x03];
   uiRange -= uiRangeLPS;
+  if (pDecEngine->iBitsLeft < 0) {
```

Comment:
it is not appropriate to add judgment in this kernel function in terms of efficiency. Could you please find a better place to prevent branch here?